exonum/src/proto/push_proto_files.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# This script copies protobuf specification files from exonum crates and pushes them as a separate new branch to the
+# repo that could be used by various Exonum clients. The branch can then be used as a source for a pull request.
+#
+# This script is intended to be used by Exonum developers in order to update Exonum clients with stable changes in the
+# proto files. No need to execute this script for any intermediate changes.
+
+set -eu -o pipefail
+
+# prints a section header
+function header() {
+    local title=$1
+    local rest="========================================================================"
+    echo
+    echo "===[ ${title} ]${rest:${#title}}"
+    echo
+}
+
+PROTO_REPO_URI="https://github.com/exonum/exonum-proto.git"
```

Comment:
`exonum-proto` repo name collides with an `exonum` component [`exonum-proto`](https://github.com/exonum/exonum/blob/dynamic_services/components/proto/Cargo.toml).

Reply:
Is this really that bad? I mean, if I get things right this can only mislead the one who's interesting in the sources of the `exonum-proto` component. But is it really possible scenario WRT the sources are already in (almost) single place? If so - should it be renamed to, for instance, `exonum-protobuf`? I feel like just `exonum/proto` is also not an option.

Reply:
`exonum-proto-sources` maybe?

Reply:
Renamed